pkgs/tools/networking/zerotierone/default.nix

```diff
@@ -1,15 +1,39 @@
-{ lib, stdenv, buildPackages, fetchFromGitHub, openssl, lzo, zlib, iproute2, ronn }:
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+, fetchurl
 
-stdenv.mkDerivation rec {
+, buildPackages
+, iproute2
+, lzo
+, openssl
+, pkg-config
+, ronn
+, zlib
+}:
+
+let
   pname = "zerotierone";
-  version = "1.8.4";
+  version = "1.8.9";
+
+  owner = "zerotier";
+  repo = "ZeroTierOne";
 
   src = fetchFromGitHub {
-    owner = "zerotier";
-    repo = "ZeroTierOne";
+    inherit owner repo;
```

Comment:
Any reason to move them out? I'd just leave them in here.

Reply:
> Any reason to move them out?

they were used in several places at some point, but now indeed they are not